classes/Tutor.php

```diff
@@ -928,18 +928,23 @@ public static function create_database() {
 		$coupons_table = "CREATE TABLE {$wpdb->prefix}tutor_coupons (
 			id BIGINT(20) UNSIGNED NOT NULL AUTO_INCREMENT,
 			coupon_status VARCHAR(50),
+			coupon_type VARCHAR(100) DEFAULT 'code', -- coupon type 'code' or 'automatic'
 			coupon_code VARCHAR(50) NOT NULL,
 			coupon_title VARCHAR(255) NOT NULL,
 			coupon_description TEXT,
 			discount_type ENUM('percentage', 'flat') NOT NULL,
 			discount_amount DECIMAL(13, 2) NOT NULL,
-			total_usage_limit INT(10) UNSIGNED,
-			per_user_usage_limit INT(10) UNSIGNED,
```

Comment:
Why removed?

Reply:
these are based on the design
there is not user limit in the design
only a check mark to make the user usage limit by 1